spec/pom.xml

```diff
@@ -68,28 +68,53 @@
                             <goal>process-asciidoc</goal>
                         </goals>
                         <configuration>
+                            <sourceDocumentName>jakarta-data.adoc</sourceDocumentName>
                             <backend>pdf</backend>
                             <outputFile>${gen-doc-dir}/jakarta-data-${project.version}.pdf</outputFile>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>generate-pdf-doc-method-query</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>process-asciidoc</goal>
+                        </goals>
+                        <configuration>
+                            <sourceDocumentName>method-query.asciidoc</sourceDocumentName>
+                            <backend>pdf</backend>
+                            <outputFile>${gen-doc-dir}/method-query-${project.version}.pdf</outputFile>
```

Comment:
Given that the file name is likely to be externally visible, we should give some thought to what to call it, such as whether to prefix it with jakarta or jakarta-data and whether to make it more explicitly named after Query by Method Name, such as query-by-method-name. But if we do either of these, the name would get longer. I don't know what is best so I thought I'd point it out.

Reply:
Yes. I was not sure either, and I wanted to see whachall thought about it.

Reply:
`jakarta-method-name-query.pdf`?

Reply:
OK, with the thumbs up from Nathan, I've made that change to the file name.